start_bs.py

```diff
@@ -3,7 +3,9 @@
 from ophyd import *
 from ophyd.mca import (Mercury1, SoftDXPTrigger)
 from ophyd import Device, EpicsMotor, EpicsSignal, EpicsSignalRO
-from zebra import Zebra
+from mxtools.zebra import Zebra
```

Comment:
I found this in the AMX collection - is that where we are importing it from here? If so, will this work sensibly if this file is run by FMX?

Reply:
We run LSDC using a Conda environment that contains multiple dependencies, like how we are doing the FMX profile_collection. We will need to add mxtools to the new environment. Ideally, we would make LSDC pip installable, but that is a while away. Until then, we have to remember that mxtools needs to be added to the MX environment.

Also, having mxtools available as a Conda package will make it easier to integrate into future MX Conda environments.

Reply:
Looks like I answered a slightly different question - I put mxtools into NSLS-II-AMX because it's MX-specific (so shouldn't be in NSLS-II), but the NSLS-II-MX is used for processing software. We should be able to use it despite it being in the "wrong" organization.